rocks-lib/Cargo.toml

```diff
@@ -15,9 +15,9 @@ git2 = "0.19.0"
 html-escape = "0.2.13"
 httpdate = "1.0.3"
 itertools = "0.13.0"
-mlua = { version = "0.10.0", features = ["luajit52", "serialize", "send"] }
+mlua = { git = "https://github.com/vhyrro/mlua", features = ["luajit52", "serialize", "send"] }
```

Comment:
question: is this change needed for this feature? Or is it left over from the `rocks add` feature?
If it's needed, we'll have to add

```nix
outputHashes = {
  "mlua-0.10.0" = "<hash>";
};
```
to the cargo lockfile import in the nix build expression.

Reply:
It's needed in this PR - I'm wondering whether I could refactor the code to stop this from being a requirement though. I'll look into it in 2 hours.